typing speed test/main.py

```diff
@@ -0,0 +1,178 @@
+import sys
+import os
+import time
+import random
+import pygame
+from pygame.locals import *
+
+class Test:
+
+    def __init__(self):
+        self.color_heading = (255, 213, 102)
+        self.color_text = (255, 0, 0)
+        self.color_results = (255, 70, 70)
+        self.w = 750
+        self.h = 490
+        self.reset = True
+        self.wpm = 0
+        self.end = False
+        self.active = False
+        self.input_text = ''
+        self.word = ''
+        self.results = 'Time:0 Accuracy:0 % WPM:0 '
+        self.start_time = 0
+        self.overall_time = 0
+        self.accuracy = '0%'
+
+        pygame.init()
+        self.image_open = pygame.image.load(os.path.abspath("/home/abhishek/Desktop/GITHUB/SWOC/Amazing-Python-Scripts/typing speed test/typing_speed.png"))

```

Comment:
The image path should be edited accordingly when the project is cloned on the local machine. Please do mention this here as in-code comment and also mention it in the README.